apps/docker/README.md

```
@@ -39,3 +39,50 @@ $ docker compose up -d --build
+
+`acore/ac-wotlk-db` is a MySQL 8.4 server **with the AzerothCore database
+already imported** (`acore_auth`, `acore_world`, `acore_characters`, including
+Eluna/mod-ale SQL to match the published `worldserver`/`authserver` images). It
+boots instantly — there is no multi-minute import on first run. It is meant for
+quick-start, dev, ephemeral test fixtures, and demos, **not production**.
+
+The image *is* the database server; run it in place of the `mysql:8.4` +
+`ac-db-import` pair:
+
+```console
+$ docker run -d --name acdb -p 3306:3306 acore/ac-wotlk-db:master
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)